diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/smb.conf/50printers mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/smb.conf/50printers
--- e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/smb.conf/50printers	2003-11-25 11:56:01.000000000 -0500
+++ mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/smb.conf/50printers	2006-04-08 20:30:41.000000000 -0400
@@ -2,7 +2,7 @@
 # NOTE: If you have a BSD-style print system there is no need to 
 # specifically define each individual printer
 # Set public = yes to allow user 'guest account' to print
-}
+    $OUT = <<HERE;
 [printers]
 comment = All Printers
 path = /var/spool/samba
@@ -10,4 +10,7 @@
 guest ok = no
 writable = no
 printable = yes
-use client driver = yes
+HERE
+    $OUT .= "use client driver = ";
+    $OUT .= $smb{UseClientDriver} || "yes";
+}
