diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/atalk/papd.conf/20printers mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/atalk/papd.conf/20printers
--- e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/atalk/papd.conf/20printers	2006-03-15 23:41:21.000000000 -0700
+++ mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/templates/etc/atalk/papd.conf/20printers	1969-12-31 17:00:00.000000000 -0700
@@ -1,16 +0,0 @@
-{
-    # process all printers
-
-    use esmith::AccountsDB;
-    my $adb = esmith::AccountsDB->open_ro();
-
-    $OUT = "";
-
-    foreach my $printer ($adb->printers)
-    {
-        $OUT .= "\n";
-        $OUT .= $printer->prop('Description') . ":\\\n";
-        $OUT .= "\t:pr=|/usr/bin/lpr -P" . $printer->key . ":\n";
-    }
-}
-
