diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.13.3/createlinks mezzanine_patched_e-smith-LPRng-1.13.3/createlinks
--- e-smith-LPRng-1.13.3/createlinks	2005-12-20 14:09:47.557675748 +1100
+++ mezzanine_patched_e-smith-LPRng-1.13.3/createlinks	2005-12-20 14:09:33.642096903 +1100
@@ -59,4 +59,7 @@
 # Make service statup symlinks
 safe_symlink("../daemontools", "root/etc/rc.d/init.d/supervise/lpd");
 safe_symlink("/etc/rc.d/init.d/e-smith-service", "root/etc/rc.d/rc7.d/S60lpd");
+safe_symlink("/etc/rc.d/init.d/e-smith-service", "root/etc/rc.d/rc0.d/K60lpd");
+safe_symlink("/etc/rc.d/init.d/e-smith-service", "root/etc/rc.d/rc1.d/K60lpd");
+safe_symlink("/etc/rc.d/init.d/e-smith-service", "root/etc/rc.d/rc6.d/K60lpd");
 safe_symlink("/var/service/lpd", "root/service/lpd");
