diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/printers mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/printers
--- e-smith-LPRng-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/printers	2008-02-13 23:12:28.000000000 +1100
+++ mezzanine_patched_e-smith-LPRng-1.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/printers	2008-02-13 23:11:28.000000000 +1100
@@ -36,10 +36,6 @@
       <trans>Remote address</trans>
     </entry>
     <entry>
-      <base>ACTION</base>
-      <trans>Action</trans>
-    </entry>
-     <entry>
       <base>REMOTE_NAME</base>
       <trans>Remote name</trans>
     </entry>
@@ -166,10 +162,6 @@
       <trans>Remove printer</trans>
     </entry>
     <entry>
-      <base>REMOVE</base>
-      <trans>Remove</trans>
-    </entry>
-    <entry>
       <base>ABOUT_TO_REMOVE</base>
       <trans>You are about to remove the printer: </trans>
     </entry>
@@ -197,12 +189,6 @@
         <base>MUST_BE_VALID_HOSTNAME_OR_IP</base>
         <trans>Must be a valid hostname or IP number</trans>
     </entry>
-
-    <entry>
-	<base>Configuration</base>
-	<trans>Configuration</trans>
-    </entry>
-
     <entry>
 	<base>Printers</base>
 	<trans>Printers</trans>
