diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-create mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-create
--- e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-create	2002-04-11 08:10:44.000000000 -0600
+++ mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-create	2005-07-17 23:04:15.195959936 -0600
@@ -30,8 +30,8 @@
 my %conf;
 tie %conf, 'esmith::config';
 
-my %accounts;
-tie %accounts, 'esmith::config', '/home/e-smith/accounts';
+use esmith::AccountsDB;
+my $adb = esmith::AccountsDB->open_ro();
 
 my $event = $ARGV [0];
 my $printerName = $ARGV [1];
@@ -42,9 +42,8 @@
 
 die "Printername argument missing." unless defined ($printerName);
 
-my ($type, %properties ) = db_get(\%accounts, $printerName);
-
-if ($type ne 'printer')
+my $p = $adb->get($printerName) || undef;
+if (not defined $p || $p->prop('type') ne 'printer')
 {
     die "Invalid printer $printerName; create printer failed.\n";
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-delete mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-delete
--- e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-delete	2002-04-11 08:10:44.000000000 -0600
+++ mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/events/actions/printer-delete	2005-07-17 23:04:32.757290208 -0600
@@ -30,8 +30,8 @@
 my %conf;
 tie %conf, 'esmith::config';
 
-my %accounts;
-tie %accounts, 'esmith::config', '/home/e-smith/accounts';
+use esmith::AccountsDB;
+my $adb = esmith::AccountsDB->open_ro();
 
 my $event = $ARGV [0];
 my $printerName = $ARGV [1];
@@ -42,10 +42,9 @@
 
 die "printerName argument missing." unless defined ($printerName);
 
-my ($type, %properties ) = db_get(\%accounts, $printerName);
-
+my $p = $adb->get($printerName);
 die "Invalid printer $printerName; delete printer failed.\n" unless
-    ($type eq 'printer-deleted');
+    (defined $p && $p->prop('type') eq 'printer-deleted');
 
 system("/bin/rm", "-rf", "/var/spool/lpd/$printerName") == 0
     or die "Could not delete spool directory '/var/spool/lpd/$printerName';" .
diff -Nur -x '*.orig' -x '*.rej' e-smith-LPRng-1.13.1/root/etc/e-smith/templates/etc/printcap/05printer mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/templates/etc/printcap/05printer
--- e-smith-LPRng-1.13.1/root/etc/e-smith/templates/etc/printcap/05printer	2002-04-11 08:10:44.000000000 -0600
+++ mezzanine_patched_e-smith-LPRng-1.13.1/root/etc/e-smith/templates/etc/printcap/05printer	2005-07-17 23:00:52.359215480 -0600
@@ -1,35 +1,31 @@
 {
-    use esmith::db
-
     # process all accounts
 
-    my %accounts;
-    tie %accounts, 'esmith::config', '/home/e-smith/accounts';
+    use esmith::AccountsDB;
+    my $adb = esmith::AccountsDB->open_ro();
 
-    my @printers = grep { db_get_type(\%accounts, $_) eq 'printer' }
-			keys %accounts;
     $OUT = "";
 
-    foreach my $printer ( @printers )
+    foreach my $printer ( $adb->printers )
     {
         $OUT .= "\n";
         $OUT .= $printer; 
-        $OUT .= "|" . db_get_prop(\%accounts, $printer, 'Description') . ":\\\n";
+        $OUT .= "|" . $printer->prop('Description') . ":\\\n";
         $OUT .= "\t:server:\\\n";
         $OUT .= "\t:mx#0:\\\n";
         $OUT .= "\t:sh:\\\n";
         $OUT .= "\t:ff_separator=false:\\\n";
-        $OUT .= "\t:sd=/var/spool/lpd/$printer:\\\n";
+        $OUT .= "\t:sd=/var/spool/lpd/" . $printer->key . ":\\\n";
 
-        if (db_get_prop(\%accounts, $printer, 'Location') eq 'remote') 
+        if ($printer->prop('Location') eq 'remote') 
         {
-	    $OUT .= "\t:rm=" . db_get_prop(\%accounts, $printer, 'Address') . ":\\\n";
+	    $OUT .= "\t:rm=" . $printer->prop('Address') . ":\\\n";
 
 	    # no trailing backslash for last entry
 
-	    if (db_get_prop(\%accounts, $printer, 'RemoteName'))
+	    if (defined $printer->prop('RemoteName'))
 	    {
-	        $OUT .= "\t:rp=" . db_get_prop(\%accounts, $printer, 'RemoteName') . ":\n";
+	        $OUT .= "\t:rp=" . $printer->prop('RemoteName') . ":\n";
 	    }
 	    else
 	    {
@@ -39,7 +35,7 @@
         else
         {
 	    # no trailing backslash for last entry
-	    $OUT .= "\t:lp=/dev/" . db_get_prop(\%accounts, $printer, 'Location') . ":\n";
+	    $OUT .= "\t:lp=/dev/" . $printer->prop('Location') . ":\n";
         }
     }
 
